CliClient/tests/MdToMd.js

```diff
@@ -0,0 +1,26 @@
+const mdImporterService = require('lib/services/InteropService_Importer_Md');
+const Note = require('lib/models/Note.js');
+const { setupDatabaseAndSynchronizer, switchClient } = require('test-utils.js');
+
+const importer = new mdImporterService();
+
+const filePath = `${__dirname}/md_to_md/sample.md`;
+
+const tagNonExistentFile = '![does not exist](does_not_exist.png)';
+
+describe('InteropService_Importer_Md', function() {
+	describe('importLocalImages method', function() {
```

Comment:
Please could you remove this level (in general we just have one "describe()" command and several "it()").

Reply:
done